### DISCUSSION TITLE
4000 uncategorised item

### DISCUSSION THREAD

**Discussion by Mreco99** [actualbudget/releases#252] [Feature ideas]
Hi
Because I started AB by importing x years of data, and that data import didn't import categories. AB said I had 4000+ uncategorised items (in red at the top of the screen). Well I wasn't going to retrospectively sort all that out for the budget. But I did go back to the beginning of the year and categorize those items (so I could get a budget up and running. 
But now, I have this continual message at the top of the screen. "4013 uncategorised transactions"

Its not a disaster, but I wish is wasn't there and I could somehow sort this out, so I can actually reliably see if ive missed categorising something. How about telling the budget not to bother with anything older than X date. or mass filling transaction cats with a "N/A" cat or some other bulk idea.

**Comment by Kidglove57** [comment]
+1 for this!! I experienced a similar thing when I joined and ended up having to make a fresh start at 1 Jan 2020 (with a bit of retrospective categorisation).
